redash/tasks.py

```diff
@@ -2,6 +2,11 @@
 import logging
 from flask.ext.mail import Message
 import redis
+import hipchat
+import requests
+import json
+from bson import json_util
```

Comment:
I prefer to keep the structure as it was before (alert in a field of its own, instead of top level). Just don't use `bson`'s `json_util` but rather re:dash's utilities for dumping json.